components/scream/src/physics/p3/p3_functions_f90.cpp

```diff
@@ -1812,11 +1812,11 @@ void calc_rime_density_f(
   Real local_rho_qm_cloud = *rho_qm_cloud_;
 
   Kokkos::parallel_for(1, KOKKOS_LAMBDA(const Int&) {
-      typename P3F::Spack t(t_), rhofaci(rhofaci_), table_val_qi_fallspd(table_val_qi_fallspd_), acn(acn_),
+      typename P3F::Spack T_atm(t_atm_), rhofaci(rhofaci_), table_val_qi_fallspd(table_val_qi_fallspd_), acn(acn_),
```

Comment:
Here's an example of the "T" case change being used back-to-back, but this isn't a pattern followed by other variables.

Reply:
Yeah, let's use the underscore rather than the "T" case to indicate the local vs argument variable.